include/dav1d/headers.rs

```diff
@@ -543,38 +543,67 @@ pub(crate) const RAV1D_TRC_UNKNOWN: Rav1dTransferCharacteristics = DAV1D_TRC_UNK
 pub(crate) const _RAV1D_TRC_BT709: Rav1dTransferCharacteristics = DAV1D_TRC_BT709;
 
 pub type Dav1dMatrixCoefficients = c_uint;
+pub const DAV1D_MC_IDENTITY: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::IDENTITY.to_dav1d();
+pub const DAV1D_MC_BT709: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::BT709.to_dav1d();
+pub const DAV1D_MC_UNKNOWN: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::UNKNOWN.to_dav1d();
+pub const DAV1D_MC_FCC: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::FCC.to_dav1d();
+pub const DAV1D_MC_BT470BG: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::BT470BG.to_dav1d();
+pub const DAV1D_MC_BT601: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::BT601.to_dav1d();
+pub const DAV1D_MC_SMPTE240: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::SMPTE240.to_dav1d();
+pub const DAV1D_MC_SMPTE_YCGCO: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::SMPTE_YCGCO.to_dav1d();
+pub const DAV1D_MC_BT2020_NCL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::BT2020_CL.to_dav1d();
+pub const DAV1D_MC_BT2020_CL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::BT2020_CL.to_dav1d();
+pub const DAV1D_MC_SMPTE2085: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::SMPTE2085.to_dav1d();
+pub const DAV1D_MC_CHROMAT_NCL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::CHROMAT_NCL.to_dav1d();
+pub const DAV1D_MC_CHROMAT_CL: Dav1dMatrixCoefficients =
+    Rav1dMatrixCoefficients::CHROMAT_CL.to_dav1d();
+pub const DAV1D_MC_ICTCP: Dav1dMatrixCoefficients = Rav1dMatrixCoefficients::ICTCP.to_dav1d();
+// dav1d defines this value as 255, but the specficiation defines it as 3
 pub const DAV1D_MC_RESERVED: Dav1dMatrixCoefficients = 255;
```

Comment:
is there someone we can ask about this? the value here needs to be 255 I think to maintain dav1d compatability (although this value is never used in the codebase)

Reply:
in practice, at the moment, neither `3` or `255` are used by the tests